conf/modules.config

```diff
@@ -554,7 +554,7 @@ if (!params.skip_alignment && params.aligner == 'star_salmon') {
 
     if (params.with_umi) {
         process {
-            withName: 'NFCORE_RNASEQ:RNASEQ:SAMTOOLS_SORT' {
+            withName: '.*:.*:SAMTOOLS_SORT' {
```

Comment:
This overrides the settings from `'.*:.*:.*:BAM_SORT_SAMTOOLS:SAMTOOLS_SORT'` above